aruna/api/internal/v1/notification.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+import "google/api/visibility.proto";
+package aruna.api.internal.v1;
+option go_package = "github.com/ArunaStorage/go-api/aruna/api/internal/v1";
+
+service InternalEventService {
+    option (google.api.api_visibility).restriction = "INTERNAL";
+    rpc RegisterEvent(RegisterEventRequest) returns (RegisterEventResponse) {}
+}
+
+message RegisterEventRequest{
+    EventResource event_resource = 1;
+    string resource_id = 2;
+    EventType event_type = 3;
+}
+
+enum EventResource {
```

Comment:
Maybe this should also include `Projects`

Reply:
Solved by [47a9bf7](https://github.com/ArunaStorage/ArunaAPI/pull/39/commits/47a9bf74d43af884769f58a39f9208352c11526d) and [72facaf](https://github.com/ArunaStorage/ArunaAPI/pull/39/commits/72facaf08a2ec5a39366389dad1fa1ed4cd0c120)